website/content/en/highlights/2021-12-15-splunk-hec-improvements.md

```diff
@@ -0,0 +1,37 @@
+---
+date: "2021-12-15"
+title: "Splunk HEC Improvements"
+description: "Improved compatibility with Splunk HTTP Event Collector"
+authors: ["barieom"]
+pr_numbers: [10261, 10135]
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+We're excited to share that we've made a couple improvements to Vector's support
+for Splunk HEC; Vector now supports Splunk HEC indexer acknowledgements and
```

Comment:
I might add some links to the Splunk HEC docs for people who are familiar with Splunk, but not the HEC format.